dependencies/ruby/build_ruby.bat.in

```diff
@@ -15,8 +15,8 @@ echo "***** Building Ruby Release Build"
 echo "****"
 
 cd ..\Ruby-release
-set CL = /MP
-set CFLAGS = MD -Zi -W2 -wd4996 -we4028 -we4142 -O2sy-  -Zm600 
+set CL=/MP
+set CFLAGS=-MD -Zi -W2 -wd4996 -we4028 -we4142 -O2sy-  -Zm600 
```

Comment:
We should put a comment in here that no spaces are allowed after the `=` sign

Reply:
Good call

Reply:
Done.